### DISCUSSION TITLE
Looks like the newest version of spacypdfreader doesn't work properly with installation

### DISCUSSION THREAD

**Discussion by junaiderade** [explosion/spaCy#12651] [Help: Installation] [answered]
i have the old version from jan on another computer it works great. i tried installing the latest on my personal computer and its not working.

import spacy
import spacypdfreader
from spacypdfreader import pdf_reader
​
nlp = spacy.load('en_core_web_sm')
doc = pdf_reader('My Notes - LeetCode.pdf', nlp)
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
/var/folders/rz/nyny__0570x556y5nyjcgppc0000gn/T/ipykernel_20275/4254746689.py in <module>
      1 import spacy
      2 import spacypdfreader
----> 3 from spacypdfreader import pdf_reader
      4 
      5 nlp = spacy.load('en_core_web_sm')

ImportError: cannot import name 'pdf_reader' from 'spacypdfreader' (/Users/junaidmohamed/opt/anaconda3/lib/python3.9/site-packages/spacypdfreader/__init__.py)

**Comment by vinbo8** [comment, marked as answer]
Thanks for the report! We've updated the documentation in #12661, you'd want to replace `from spacypdfreader import pdf_reader` with `from spacypdfreader.spacypdfreader import pdf_reader`.
